WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -93,6 +93,7 @@ class WordPressAppDelegate: UIResponder, UIApplicationDelegate {
         AccountService(managedObjectContext: mainContext).restoreDisassociatedAccountIfNecessary()
 
         customizeAppearance()
+        configureAnalytics()
```

Comment:
I need analytics to be initialized before we start the re-authentication flow, so that we can track analytics throughout said flow.